src/App.js

```diff
@@ -1,11 +1,12 @@
+import ProductsComponent from "./Products";
+
 function App() {
   return (
     <div>
       <h1>Tech shop</h1>
-      {/* Render the product component below */}
-
+      <ProductsComponent />
```

Comment:
Hello, I would avoid adding Component suffix, When we use this the first letter uppercase syntax in jsx it is implied that they are components and not predefined elements from html.